share/spice/indego_bikeshare_locations/indego_bikeshare_locations.js

```diff
@@ -0,0 +1,78 @@
+(function(env) {
+    "use strict";
+
+    env.ddg_spice_indego_bikeshare_locations = function(api_result) {
+        if (!api_result || !api_result.features) {
+            return Spice.failed('indego_bikeshare_locations');
+        }
+
+   
+
+        DDG.require('moment.js', function() {
+            DDG.require('maps', function() {
+             moment.locale('en', {
+                    relativeTime : {
+                        past:   "%s ago",
```

Comment:
In #2151 I replaced `a` and `an` with `1` so it's a little more "proper" sounding. "a sec ago" sounds a bit too casual and a little vague.